arrow-libs/core/arrow-core-retrofit/build.gradle

```
@@ -1,32 +1,18 @@
 plugins {
-    id "org.jetbrains.kotlin.multiplatform"
+    id "org.jetbrains.kotlin.jvm"
     id "org.jlleitschuh.gradle.ktlint"
 }
 
-apply from: "$SUB_PROJECT_MPP"
+apply from: "$SUB_PROJECT"
 apply from: "$TEST_COVERAGE"
 apply from: "$DOC_CREATION"
 apply from: "$PUBLICATION_MPP"
 apply from: "$ANIMALSNIFFER"
 
-kotlin {
-    sourceSets {
-        jvmMain {
-          dependencies {
-            compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
-            compileOnly project(":arrow-core")
-            compileOnly "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
-          }
-        }
-
-        jvmTest {
-            dependencies {
-              compileOnly "org.jetbrains.kotlin:kotlin-reflect:$KOTLIN_VERSION"
-              runtimeOnly "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
-              implementation project(":arrow-core-test")
-              implementation "com.squareup.retrofit2:converter-gson:$RETROFIT_VERSION"
-              implementation "com.squareup.okhttp3:mockwebserver:$MOCKWEBSERVER_VERSION"
-            }
-        }
-    }
+dependencies {
+    api project(":arrow-core")
+    api project(":arrow-continuations")
+    api "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
+    api "io.kotlintest:kotlintest-runner-junit5:$KOTLIN_TEST_VERSION", excludeArrow
+    testRuntimeOnly "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
```

```suggestion
dependencies {
    compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
    compileOnly project(":arrow-core")
    compileOnly "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
    testCompileOnly "org.jetbrains.kotlin:kotlin-reflect:$KOTLIN_VERSION"
    testRuntimeOnly "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
    testImplementation project(":arrow-core-test")
    testImplementation "com.squareup.retrofit2:converter-gson:$RETROFIT_VERSION"
    testImplementation "com.squareup.okhttp3:mockwebserver:$MOCKWEBSERVER_VERSION"
```

That's the previous configuration for `arrow-core-retrofit`. Maybe it's working with those dependencies because the code is missing in the previous location.

The source code should be moved as well :raised_hands: